canonical-websites/www.ubuntu.com
# Upgrade to v2.7.0 of canonical-webteam tooling

This may be a little gung-ho, but here I'm using version 2.7.0 from https://github.com/canonical-webteam/generator-canonical-webteam/pull/60 of the run script, so that dependency checking will be much more robust.

QA
--

Check run script works as expected.

``` bash
./run clean
./run
./run build
... etc
```

Check everything works.
